content/blog/2024-05-23-linebender-roadmap.md

```diff
@@ -0,0 +1,248 @@
++++
+title = "May-July roadmap for Linebender"
+authors = ["Olivier Faure"]
+date = "2024-05-23 18:00:00"
++++
+
+On May 7 and 8, 2024, folks from the Linebender team went to [the RustNL conference](https://2024.rustnl.org/) for two days, followed by two days of Unconference.
+
+By that point, we all felt fairly exhausted, but Raph Levien being the cruel taskmaster that he is, he still made us work one extra day, and so we came together in our hotel's conference room to discuss the Linebender project, and what we wanted from it.
+
+We'd initially thought of that this day would be an informal day of unstructured chatting about the ecosystem, but it fairly quickly turned into a semi-formal planning exercise, where we tried to figure out a roadmap for the next few months.
+
+What follows is a comprehensive summary of what exactly we're planning to do.
+We've tried to focus on blockers, where a missing feature in one project is stalling progress on other projects, because those are the hardest barriers to coordination, and we wanted to take advantage of being in the same room together to knock them down.
+
```

Comment:
I think we should also have a comment about supporting community contribution both in these areas and beyond them (maybe? pointing to the new long term roadmap?)

Reply:
I'll pass. I think the intro and conclusion are already too wordy.